test/fixtures/build-withTsconfig/tsconfig.json

```
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "module": "ESNext",
+    "lib": ["dom", "esnext"],
+    "declaration": true,
+    "declarationDir": "typings",
+    "declarationMap": true,
+    "sourceMap": true,
+    "rootDir": "./src",
```

```suggestion
    "rootDir": "./",
```

For consistency with the other fixtures. `"./"` is actually incorrect, according to #488 , but I'd rather make all the changes to `"./src"` in one place.
This one change made me very confused as to why the declarationMap was output correctly here but not in the other fixtures when working on #488 (see https://github.com/jaredpalmer/tsdx/issues/479#issuecomment-581721483 and the comment before it)